mycore-mods/src/main/java/org/mycore/mods/MCRMODSLinksEventHandler.java

```diff
@@ -41,9 +40,6 @@
 public class MCRMODSLinksEventHandler extends MCREventHandlerBase {
 
     public static final String INDEX_ALL_CHILDREN_PROPERTY_NAME = "MCR.MODS.LinksEventHandler.IndexAllChildren";
```

Comment:
This constant and the value in mycore.properties are not used anymore and can be deleted